ladybug/src/main/resources/springIbisDebuggerAdvice.xml

```diff
@@ -11,7 +11,7 @@
 	>
 
 	<bean
-		name="ibisDebuggerAdvice,threadLifeCycleEventListener"
+		name="ibisDebuggerAdvice,threadLifeCycleEventListener,xmlDebugger"
```

Comment:
Moet dit?

Reply:
Dat is nodig zodat xmlDebugger geinjecteerd wordt in Pipes en Senders die hem willen gebruiken.

Reply:
Ik zie het, de IbisDebuggerAdvice is een behoorlijke knaap aan het worden zo :)